.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. \
+List any dependencies that are required for this change.
+
+Fixes #(issue)
+
+## Type Of Change
+
+Please check the relevant options.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. \
+Provide instructions so we can reproduce. \
+Please also list any relevant details for your test configuration.
+
+- Run command `:mycommand`
+- Check logs
+- ...
+
+## Checklist:
+
+- [ ] I read the [contributing](https://github.com/ChristianChiarulli/LunarVim/blob/rolling/CONTRIBUTING.md) guide
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```

Comment:
I'm not sure, but should we write `commented on my code` or `commented my code`
sorry If I'm being picky :smirk_cat:

Reply:
I'm not a native English speaker, so these details are out of my league :sweat_smile:

Reply:
me  neither @ChristianChiarulli  a bit help, please 🥺

Reply:
I think the correct way here is: I have commented my code.